install.sh

```diff
@@ -340,9 +348,10 @@ add_trdl_werf_repo() {
   declare werf_tuf_repo_url="$1"
   declare werf_tuf_repo_root_version="$2"
   declare werf_tuf_repo_root_sha="$3"
+  declare user="$4"
 
   log::info "Adding werf repo to trdl."
-  "$HOME/bin/trdl" add werf "$werf_tuf_repo_url" "$werf_tuf_repo_root_version" "$werf_tuf_repo_root_sha" || abort "Can't add \"werf\" repo to trdl."
+  run_as_user $user "$HOME/bin/trdl add werf $werf_tuf_repo_url $werf_tuf_repo_root_version $werf_tuf_repo_root_sha" || abort "Can't add \"werf\" repo to trdl."
```

Comment:
Just realised `$user` mostly is not escaped with double quotes in the script, please do.